src/NUnitFramework/framework/Internal/ConstraintUtils.cs

```diff
@@ -32,16 +31,18 @@ namespace NUnit.Framework.Internal
     internal static class ConstraintUtils
     {
         /// <summary>
-        /// Require that the provided object is actually of the type required,
-        /// and that it is not null.
+        /// Requires that the provided object is actually of the type required.
         /// </summary>
         /// <param name="actual">The object to verify.</param>
         /// <param name="paramName">Name of the parameter as passed into the checking method.</param>
-        /// <typeparam name="T">The type to require.</typeparam>  
-        /// <returns>A properly typed object, or throws. Never null.</returns>
-        public static T RequireActual<T>(object actual, string paramName)
+        /// <param name="allowNull">
+        /// If <typeparamref name="T"/> can be null, returns null rather than throwing when <paramref name="actual"/> is null.
```

Comment:
Perhaps we should state that this only is the case when `allowNull` is true, but perhaps this is implicit?

Reply:
I'm always interested in making things clearer. `If <see langword="true"/> and`?

Reply:
Sounds good